lib/ts/config.ts

```diff
@@ -546,12 +580,35 @@ export async function getQuestions(flags: UserFlags) {
             name: "backend",
             type: "list",
             message: "Choose a backend framework (Visit our documentation for integration with other frameworks):",
-            choices: mapOptionsToChoices(await getBackendOptions(flags)),
+            choices: mapOptionsToChoices(await getBackendOptions()),
             when: (answers: Answers) => {
                 // If shouldSkipBackendQuestion returns true we want to return false from here
                 return !shouldSkipBackendQuestion(answers, flags);
             },
         },
+        {
+            name: "backendNodeJS",
+            type: "list",
+            message: "Choose a Node.js framework:",
+            choices: mapOptionsToChoices(await getNodeJSOptions(flags)),
+            when: (answers: Answers) => {
+                if (flags.backend !== undefined && flags.backend !== "nodejs") {
+                    return false;
+                }
+                if (
+                    (flags.frontend !== undefined && flags.frontend.startsWith("next")) ||
+                    (answers.frontend !== undefined && answers.frontend.startsWith("next"))
+                ) {
+                    // This means that they want to use nextjs fullstack
```

Comment:
what about other types of fe like astro / remix etc?

Reply:
is this checking required? I had just give this checking because it was also in backendPython, but as we are checking 
```js                
if (answers.backend !== "node" && flags.backend !== "node")
         return false;
 }
```
it will return false only if the backend is not node. is this checking required?

Reply:
It works functionally with that code however the code needs to be better organised so that the flow is simpler.

Essentially, the backend frameworks should be part of the backend object so that if the user selects the `node` option, the code picks up the nested options from that object directly instead of keeping another one called `NodeJSFrameworks` etc.

Reply:
I was wrong problem will occur when someone will run something like
`npx create-supertokens-app --appname=my-app --recipe=emailpassword --frontend=remix --backend=python
` it will ask for which backend framework in python , but for `--frontend=next` the option to choose backend framework will not come, updating..

Reply:
Done

Reply:
Hi @rishabhpoddar please re-review